resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -167,6 +173,11 @@ function task_install {
 <% EZBake::Config[:cli_app_files].each do |cli_app_file| -%>
     install -m 0755 <%= cli_app_file %> "${DESTDIR}${app_prefix}/cli/apps/<%= cli_app_file.sub(/^ext\/cli\//, "") %>"
 <% end -%>
+
+if [ -e "<%= EZBake::Config[:cli_defaults_file] %>" ]; then
+    install -m 0755 <%= EZBake::Config[:cli_defaults_file] %> "${DESTDIR}${app_prefix}/cli/apps/<%= EZBake::Config[:cli_defaults_file] %>"
```

Comment:
Maybe here is where we could have the `cli_defaults_file` be installed to .../cli instead of .../cli/apps - so that the cli-app script doesn't interpret it as an app?